src/main/java/utils/UserContext.java

```diff
@@ -0,0 +1,15 @@
+package utils;
+
+import domain.User;
+
+public class UserContext {
+    private static final ThreadLocal<User> thUser = new ThreadLocal<>();
```

Comment:
1. Does this refer to the fact, that final fields should be uppercase? 
2. Is this a hint to 1. and a suggestion how to correctly name a variable (instead of thUser) or is it a better option to name the class?